test/ez-navigation-hub.html

```diff
@@ -8,17 +8,34 @@
 
         <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
         <script src="../../web-component-tester/browser.js"></script>
+        <script src="../../element-matches/closest.js"></script><!-- for Edge 14 -->
 
         <link rel="import" href="../ez-navigation-hub.html">
     </head>
     <body>
 
         <test-fixture id="BasicTestFixture">
             <template>
-                <ez-navigation-hub></ez-navigation-hub>
+                <ez-navigation-hub active-zone-class="ez-active-zone" matched-link-class="ez-matched-link">
```

Comment:
if you need such attributes (I think it's not the case), don't put default values as the attribute value